README.md

```diff
@@ -42,6 +42,15 @@ Note the last 3 resources are mandatory for the the glue-compatible hive connect
 
 Don't forget to also add S3 IAM permissions for Spark to be able to fetch table data!
 
+### GCP Bigquery/GCS credentials
+
+You must provide a valid path to a GCP service account file using environment variable `GOOGLE_APPLICATION_CREDENTIALS`.
+Otherwise you have to set manually an Access Token after the Spark Context is created using
```

Comment:
About the fact of using a file, I thought @rudy told me, they will give us a service account which should use to add the credentials in VAULT and then use ENVVAULT to provide the vars as environment variables. I am not sure we are following this pattern here. Or Am I missing something?

Reply:
Yes envvault will be used here https://github.com/tinyclues/spark-executor/pull/36